src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -358,6 +360,26 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('set-primary', 'failover_group_failover')
 
+    ###############################################
+    #             sql instance-pool               #
+    ###############################################
+
+    instance_pools_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#InstancePoolsOperations.{}',
+        client_factory=get_sql_instance_pools_operations)
+    with self.command_group('sql instance-pool', instance_pools_operations, client_factory=get_sql_instance_pools_operations, is_preview=True) as g:
+        g.command('show', 'get',
```

Comment:
Please use `show_command()` for your show command.

Reply:
Done